packages/api-derive/src/staking/info.ts

```diff
@@ -63,24 +64,24 @@ function calculateUnlocking (stakingLedger: StakingLedger | undefined, eraLength
 
   // group the Unlockchunks that have the same era and sum their values
   const groupedResult = groupByEra(unlockingChunks);
-  const results = Object.entries(groupedResult).map(([eraString, value]): { value: BN; remainingBlocks: BN } => ({
-    value,
+  const results = Object.entries(groupedResult).map(([eraString, value]): DerivedUnlocking => ({
+    value: createType('Balance', value),
     remainingBlocks: remainingBlocks(createType('BlockNumber', eraString), eraLength, bestNumber)
   }));
 
   return results.length ? results : undefined;
 }
 
-function redeemableSum (stakingLedger: StakingLedger | undefined, eraLength: BN, bestNumber: BlockNumber): BN {
+function redeemableSum (stakingLedger: StakingLedger | undefined, eraLength: BN, bestNumber: BlockNumber): Balance {
   if (isUndefined(stakingLedger)) {
-    return new BN(0);
+    return createType('Balance');
   }
 
-  return stakingLedger.unlocking.reduce((total, { era, value }): BN => {
+  return createType('Balance', stakingLedger.unlocking.reduce((total, { era, value }): BN => {
     return remainingBlocks(era.unwrap(), eraLength, bestNumber).eqn(0)
       ? total.add(value.unwrap())
       : total;
-  }, new BN(0));
+  }, new BN(0)));
```

Comment:
Would be nice to remove all BN instances in the code base (except in AbstractInt codec)